ballerina/utils.bal

```diff
@@ -538,10 +538,10 @@ isolated function copyFile(http:Client httpClient, string fileId, CopyFileOption
 # 
 # + httpClient - The HTTP Client
 # + fileId - ID of the file to be updated
+# + fileResource - 'FileMetadata' can be added as a payload to change metadata
 # + optional - 'UpdateFileMetadataOptional' used to add query parameters to the request
-# + fileResource - 'File' can added as a payload to change metadata
 # + return - If successful, returns `File`. Else returns `error`
-isolated function updateFileById(http:Client httpClient, string fileId, File? fileResource = (), 
```

Comment:
This seems like a breaking change, is this intentional ?

Reply:
@ayeshLK this is just an util function consumed by the connector API. As this function is not public, won't be a breaking change IMO.

Reply:
agreed